lib/active_merchant/billing/gateways/cecabank/cecabank_json.rb

```diff
@@ -140,8 +140,23 @@ def add_stored_credentials(post, creditcard, options)
 
       def add_three_d_secure(post, options)
         params = post[:parametros] ||= {}
-
-        params[:ThreeDsResponse] = options[:three_d_secure] if options[:three_d_secure]
+        three_d_secure = options[:three_d_secure]
+        return unless three_d_secure.present?
+
+        three_d_response = {}
+        three_d_response[:exemption_type] = three_d_secure[:exemption_type]
```

Comment:
⚠️ Warning:

`exemption_type` is a field on the request to core but is not part of the values that core passes down to AM, check [this params](https://github.com/spreedly/core/blob/master/app/models/gateway.rb#L757)